sections/introduction.tex

```diff
@@ -1,23 +1,32 @@
 \section{Introduction} 
 \todo[inline]{should be twice as long as it is now}
 %% Argue briefly for the relevance of the studied area/problem — start from general and end with your concrete problem.
-A general challenge for search engines, no matter the environment, is how to acquire relevant search results from a given corpus.
-While some search engines use simple algorithms, such as finding all articles including a given search term and sorting by date, this may not always give useful results.
+
+Many news articles are produced every day and the need for searching the news is becoming more prominent and difficult task.
+A general challenge for search engines, no matter the environment, is how to acquire relevant search results from a given corpus. 
+While some engines use simple algorithms, such as finding all articles including a given search term and sorting by date, this may not always give useful results.
```

Comment:
```suggestion
While some engines use simple algorithms, such as finding all articles that include a given search term and sorting by date, this may not always give useful results.
```